RELEASE_NOTES

```diff
@@ -3,16 +3,49 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 3,344 easyconfig files, for 581 different software packages,
-32 different (compiler) toolchains and 6 software bundles.
+The latest version of easybuild-easyconfig provides 3,344 easyconfig files, for 608 different software packages,
+33 different (compiler) toolchains and 8 software bundles.
 
+v2.1.0 (April 28th 2015)
+------------------------
+
+- added example easyconfig files for 27 new software packages:
+    AFNI (#1322, #1521), BCFtools (#1492), getdp (#1518), gmsh (#1518), gtest (#1244), hanythingondemand (#1530),
+    mawk (#1369), Minimac (#815), Minimac3 (#1502), monty (#1548), Octave (#1563), pbs_python (#1530), pigz (#1036),
+    Pygments (#1536), pyhull (#1539), pymatgen (#1549), PyQt (#1322, #1521), Ray (#1494), requests (#1536),
+    seqtk (#1524), SIP (#1322, #1521), S-Lang (#1369), Spark (#1554), spglib (#1549), TINKER (#1465), tmux (#1369),
+    Xmipp (#1489)
+- added easyconfigs for new (Cray-specific) toolchains (#1538): CrayGNU, CrayIntel, CrayCCE
+    - initially supported software (using CrayGNU toolchains): CP2K, GROMACS, HPL, Python + numpy/scipy, WRF (#1558)
+- added new easyconfigs for existing toolchains: goolf/1.5.16, intel/2014.06
+- added additional easyconfigs for various supported software packages: version updates, different toolchains, ...
+    including GCC v5.1.0, OpenFOAM v2.3.1, R v3.1.3, PETSc/SLEPc v3.5.3, WIEN2k v14.1
```

Comment:
overall, this PR looks good; pity we did not add R/3.2.0 in this release (have it tested and working for several days)!